src/pages/HomePage/HomePage.js

```diff
@@ -1,8 +1,66 @@
-// HomePage.js
 import React from "react";
+import HomePageImage from './images/homepage-image_photo_noise2_scale_cropped_photo_noise2_scale.png';
+import Typography from "@mui/material/Typography";
+import "./HomePage.css"; // Import the CSS file for additional styling
+
+const imageContainerStyle = {
+  position: "relative",
+  width: "100%",
+};
+
+const imageStyle = {
+  width: "100%",
+  height: "auto",
+  objectFit: "cover",
+};
+
+const overlayStyle = {
+  position: "absolute",
+  top: 0,
+  left: 0,
+  width: "100%",
+  height: "100%",
+  backgroundColor: "rgba(0, 0, 0, 0.6)", 
+  zIndex: 1,
+};
+
+const textStyle = {
+  position: "absolute",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  fontFamily: "Work Sans, sans-serif",
+  color: "white",
+  fontSize: "65px", 
+  fontWeight: "600",
```

Comment:
```suggestion
  fontWeight: "400",
  textAlign: "center", 
```

Just making it a little less bold to match the figma design. Also, I think this text should be centered so it looks good when rescaled too.

Reply:
Got it! I think i managed to fix the conflicts--I'll make a new PR for it.